contrib/julia-config.jl

```diff
@@ -70,9 +78,9 @@ function cflags()
     arg2 = replace(includeDir(),"\\","\\\\")
     threading_def = threadingOn() ? "-DJULIA_ENABLE_THREADING=1 " : ""
     if is_unix()
-        return """$(threading_def)-fPIC -DJULIA_INIT_DIR=\\"$arg1\\" -I$arg2"""
+        return """-std=gnu99 $(threading_def)-fPIC -DJULIA_INIT_DIR=\\"$arg1\\" -I$arg2"""
     else
-        return """$(threading_def)-DJULIA_INIT_DIR=\\"$arg1\\" -I$arg2"""
+        return """-std=gnu99 $(threading_def)-DJULIA_INIT_DIR=\\"$arg1\\" -I$arg2"""
```

Comment:
I looked into why we can't use `c99` here. it looks like the difference between these options is in the compiler flags that get set (https://www.gnu.org/software/libc/manual/html_node/Feature-Test-Macros.html). In particular, to use `c99`, we would need to set `_POSIX_C_SOURCE` >= 2001xxL (for a couple of pthread structs in the libuv header). When we get rid of the uv.h installation requirement (https://github.com/JuliaLang/julia/pull/8880), we'll only need to define `_POSIX_SOURCE` somewhere (for `sigjmp_buf` visibility).